### DISCUSSION TITLE
Meeting Agenda 4/16

### DISCUSSION THREAD

**Discussion by dyc3** [dyc3/senior-design#389] [Meeting Agendas/Minutes]
1. Review tasks from previous sprint
2. Status of expo booklet
   - determine changes needed
3. Sprint Planning
4. Resolve vis not working for Victor and Mike
5. Meet with Darian
   - Booklet: How will it be bound?

**Comment by dyc3** [comment]
booklet checklist:
- [x] Remove the chapter on the weekly reports and the agile boards screenshots.
- [x] remove user interviews
- [x] remove user personas
- [x] Turn off auto glossary highlights

**Comment by moreno-michael** [comment]
Meeting minutes 4/16

1. Task Review
- Issue #1427 in progress
- Issue #1646 completed

2. Expo booklet requirements filed as tasks and listed in separate comment
- Team to comb through spec doc to point out and correct any glaring formatting and content issues

3. Sprint Planning
- This sprint will consist of mostly finishing IDE business plan and cleaning up doc, as well as finishing the legend

4. Dev environment issues
- Dev environment issues resolved for victor and mike with the docker version of visualization

5. Meet with Darian
- Add margins to top and maybe bottom of doc pages to account for printing
